core/src/autogluon/core/scheduler/reporter.py

```diff
@@ -97,6 +98,53 @@ def __repr__(self):
         return reprstr
 
 
+class MODistStatusReporter(DistStatusReporter):
```

Comment:
This is an elegant solution, but it introduces a new Reporter class. Would this survive later changes of DistStatusReporter?
I suppose you considered whether changes could be limited to the scheduler as well?